doc/install-ejabberd.md

```diff
@@ -0,0 +1,43 @@
+Install an ejabberd with synchronized credentials
+=================================================
+
+* [Home](help)
+
```

Comment:
can you write a short introduction for people who want to chat, but don't know that ejabberd might be their keyword?